<launch>

  <!-- <node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
    <param name="destination_frame" value="virtual_laser_link"/>
    <param name="cloud_destination_topic" value="/merged_cloud"/>
    <param name="scan_destination_topic" value="/scan"/>
    <param name="laserscan_topics" value ="/scan_kirmizi_laser /scan_mavi_laser" /> 
    <param name="angle_min" value="-3.1415926535"/>
    <param name="angle_max" value="3.1415926535"/>
    <param name="angle_increment" value="0.0023573737125843763"/>
    <param name="scan_time" value="0.000001"/>
    <param name="range_min" value="0.15"/>
    <param name="range_max" value="12.0"/>
  </node> -->
  
  <node name="mux_laser_scans" pkg="topic_tools" type="mux" output="screen" args="/scan /scan_kirmizi_laser /scan_mavi_laser mux:=mux_laser_scans"> </node>

</launch>